pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -393,6 +393,10 @@ type TiKVSpec struct {
 	// Config is the Configuration of tikv-servers
 	// +optional
 	Config *TiKVConfig `json:"config,omitempty"`
+
+	// InitializeStoreLabels indicates the store label set during initialization
+	// +optional
+	InitializeStoreLabels map[string]string `json:"initializeStoreLabels,omitempty"`
```

Comment:
I would prefer a shorter name, `storeLabels` or `initStorelabels`

Reply:
maybe we should this generic way to configure environments for containers

https://github.com/pingcap/tidb-operator/blob/master/pkg/apis/pingcap/v1alpha1/types.go#L711

Reply:
Setting store label by ENV is the way operator pass the store labels for tikv but not obvious and convenient for users, I still suggest exposing store labels by a certain atrributes.

Reply:
I don't think it's necessary to provide a special way for this. It's simple as configuring environments for a normal pod in Kubernetes.

Reply:
We already have too many special configurations. Anyone knowing the standard way to configure environments for containers can configure this env and any other environments for TiKV and other components. 

If we use this special field for `STORE_LABELS`, we have more things to consider, e.g. what if users set both `STORE_LABELS` env and `storeLabels` field. Should we merge them? If not, which one has a higher priority, etc. If we have a special field for this env, should we add fields for other envs if users ask?

Reply:
ok, I agree with setting store labels by env.